playbooks/besman-criticality_score-0.0.1-playbook.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+function __besman_init() {
+    __besman_echo_white "initialising"
+    export ASSESSMENT_TOOL_NAME="criticality_score"
+    export ASSESSMENT_TOOL_TYPE="criticality_score"
+    export ASSESSMENT_TOOL_VERSION="2.0.3"
+    #export BESLAB_OWNER_TYPE="Organization"
+    #export BESLAB_OWNER_NAME="Be-Secure"
+    export ASSESSMENT_TOOL_PLAYBOOK="besman-$ASSESSMENT_TOOL_TYPE-0.0.1-playbook.sh"
+
+    local steps_file_name="besman-$ASSESSMENT_TOOL_NAME-0.0.1-steps.sh"
+    export BESMAN_STEPS_FILE_PATH="$BESMAN_PLAYBOOK_DIR/$steps_file_name"
+
+    local var_array=("BESMAN_ARTIFACT_TYPE" "BESMAN_ARTIFACT_NAME" "BESMAN_ARTIFACT_VERSION" "BESMAN_ARTIFACT_URL" "BESMAN_ENV_NAME" "BESMAN_ARTIFACT_DIR" "ASSESSMENT_TOOL_NAME" "ASSESSMENT_TOOL_TYPE" "ASSESSMENT_TOOL_VERSION" "ASSESSMENT_TOOL_PLAYBOOK" "BESMAN_ASSESSMENT_DATASTORE_DIR" "BESMAN_TOOL_PATH" "BESMAN_ASSESSMENT_DATASTORE_URL" "BESMAN_LAB_TYPE" "BESMAN_LAB_NAME")
+
+    local flag=false
+    for var in "${var_array[@]}"; do
+        if [[ ! -v $var ]]; then
+
+            # read -rp "Enter value for $var:" value #remove
+            # export "$var"="$value" #remove
+            __besman_echo_yellow "$var is not set" #uncomment
+            __besman_echo_no_colour ""             #uncomment
+            flag=true                              #uncomment
+        fi
+
+    done
+
+    local dir_array=("BESMAN_ASSESSMENT_DATASTORE_DIR")
+    for dir in "${dir_array[@]}"; do
+        # Get the value of the variable with the name stored in $dir
+        dir_path="${!dir}"
+
+        if [[ ! -d $dir_path ]]; then
+            __besman_echo_red "Could not find $dir_path"
+            flag=true
+        fi
+
+    done
+
+    # [[ ! -f $BESMAN_TOOL_PATH/$ASSESSMENT_TOOL_NAME ]] && __besman_echo_red "Could not find artifact @ $BESMAN_TOOL_PATH/$ASSESSMENT_TOOL_NAME" && flag=true
```

Comment:
Replace this commented code to check if criticality_score is installed or not, and give an appropriate message

Reply:
done